site/docs/v1/tech/account-management/index.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: doc
+title: Account Management
+description: Managing Accounts - with Self Service
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
```

Comment:
Please add another page showing that users can edit their profile data and password out of the box. That's super powerful functionality.